src/api/ci/jenkins/services/jenkins-build.service.ts

```
@@ -0,0 +1,377 @@
+import { JenkinsHttpClient } from '../http/jenkins-http.client';
+import { 
+  JenkinsApiResponse, 
+  BuildOptions, 
+  BuildSearchOptions,
+  WaitForBuildOptions,
+  JenkinsBuild,
+  JenkinsJob
+} from '../types/jenkins.types';
+import { JenkinsBuildResult, JenkinsBuildTrigger } from '../enums/jenkins.enums';
```

ESLint

[error] Replace `·JenkinsBuildResult,·JenkinsBuildTrigger·}·from·'../enums/jenkins.enum` with `⏎··JenkinsBuildNotFoundError,⏎··JenkinsBuildTimeoutError,⏎··JenkinsJobNotFoundError,⏎}·from·'../errors/jenkins.error`

(prettier/prettier)